assets/onboarding/ready/index.test.js

```diff
@@ -0,0 +1,49 @@
+import { render } from '@testing-library/react';
+import { Ready } from './index';
+
+const mockStepData = {
+	admin_email: 'admin@test.local',
+	mc_url: 'http://external.local/campaign',
+	gdpr_field: 'SDCTZX3',
+};
+jest.mock( '../use-setup-wizard-step', () => ( {
+	useSetupWizardStep: jest.fn().mockImplementation( () => ( {
+		stepData: mockStepData,
+	} ) ),
+} ) );
+
+describe( '<Ready />', () => {
+	afterEach( () => {} );
```

Comment:
I think we could remove this `afterEach`, right?

Reply:
Yup, 2434b0c3